cromwell-drs-localizer/src/test/scala/drs/localizer/DrsLocalizerMainSpec.scala

```diff
@@ -138,6 +146,86 @@ class DrsLocalizerMainSpec extends AnyFlatSpec with CromwellTimeoutSpec with Mat
     actualAttempts shouldBe 4 // 1 initial attempt + 3 retries = 4 total attempts
   }
 
+  it should "retry an appropriate number of times for transient retryable access URL download failures" in {
+    var actualAttempts = 0
+
+    val drsLocalizer = new MockDrsLocalizerMain(MockDrsPaths.fakeDrsUrlWithAccessUrlResolutionOnly, fakeDownloadLocation, None) {
+      override def resolveAndDownload(downloaderFactory: DownloaderFactory): IO[DownloadResult] = {
+        actualAttempts = actualAttempts + 1
+        if (actualAttempts <= 3) {
+          IO.raiseError(new RuntimeException("testing: transient error") with TransientRetryDisposition)
```

Comment:
I see this is where the transient errors come from. Where do the regular errors come from?

Reply:
line 165